### PR TITLE
DAOS-7317 Test: Adding IOR Hard test for EC Object types.

### PR DESCRIPTION
**Pull request by ravalsam**
Quick-build: true
Test-tag: ior_hard

Signed-off-by: Samir Raval <samir.raval@intel.com>

### CODE REVIEW

**Review by mchaarawi — COMMENTED** [review]
unless im looking at the wrong place, i don't see the results for this test.
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-6131/8/artifact/Functional%20Hardware%20Large/

**Comment by ravalsam** [comment]
> unless im looking at the wrong place, i don't see the results for this test.
> https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-6131/8/artifact/Functional%20Hardware%20Large/

Sorry forgot to paste the result link. The test initially ran with ior_hard tag and later it was running with default Ci tags(Which does not has IOR Hard to be run as part of Ci).
Previous run result can be found from https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-6131/5/artifact/Functional%20Hardware%20Large/ior/ior_hard.py/

File: src/tests/ftest/ior/ior_hard.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  engines_per_host: 2
+  name: daos_server
+  servers:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      scm_class: dcpm
```
Comment (saurabhtandan) [inline src/tests/ftest/ior/ior_hard.yaml:23]: We don't want to use nvme for this test?

**Review by saurabhtandan — APPROVED** [review]
LGTM

**Review by mchaarawi — CHANGES_REQUESTED** [review]
for the read phase you are NOT supposed to pass these:
-D 300
-O stoneWallingWearOut=1

should also set
 --dfs.chunk_size=470080
not 1m to match what we actually do for io-500; unless there is a reason you are setting it to 1m.

**Comment by ravalsam** [comment]
> for the read phase you are NOT supposed to pass these:
> -D 300
> -O stoneWallingWearOut=1

I really thought it's the same. Let me remove it.

> should also set
> --dfs.chunk_size=470080
> not 1m to match what we actually do for io-500; unless there is a reason you are setting it to 1m.

Not really let me change and rerun it again

**Comment by ravalsam** [comment]
> > for the read phase you are NOT supposed to pass these:
> > -D 300
> > -O stoneWallingWearOut=1
> 
> I really thought it's the same. Let me remove it.
> 
> > should also set
> > --dfs.chunk_size=470080
> > not 1m to match what we actually do for io-500; unless there is a reason you are setting it to 1m.
> 
> Not really let me change and rerun it again

Updated the code and CI result can be found from https://build.hpdd.intel.com/job/daos-stack/job/daos/view/change-requests/job/PR-6131/11/artifact/Functional%20Hardware%20Large/ior/ior_hard.py/

**Review by mchaarawi — APPROVED** [review]

**Review by saurabhtandan — APPROVED** [review]
LGTM

**Pull request merged**